src/errorMessages.cpp

```diff
@@ -249,6 +249,11 @@ std::string ErrorMessages::WARNING_WRITE_NO_SOMA() const
     return "Warning: writing file without a soma";
 }
 
+std::string ErrorMessages::WARNING_WRITE_EMPTY_MORPHOLOGY() const
+{
+    return "Warning: attempting to write an empty morphology. Refusing to write.";
```

Comment:
_Refusing_  ? I would not be so rude with the user :)
What about something like: "Warning: aborting an attempt to write an empty morphology."  ?